core/jobs.py

```diff
@@ -945,7 +945,7 @@ def _get_continuous_computation_class(cls):
     @staticmethod
     def _get_job_queued_msec():
         """Returns the time when the job got queued, in milliseconds past the
-        Epoch.
+        epoch.
```

Comment:
Should be capitalized, since it's the Linux Epoch :)

Reply:
Fixed that!